### PR TITLE
[MAINTENANCE] Point to proper logo URI in Jinja templates

### PR DESCRIPTION
**Pull request by AFineDayFor**
🌸 
Changes proposed in this pull request:
- change logo Uri as old one is invalid, match in tests.

Previous Design Review notes:
- Don and Alex had to scramble because I bungled a thing.

### Definition of Done

- [x] I added the tests I thought were silently failing. Sorry guys.
- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code

### CODE REVIEW

File: tests/render/test_data_documentation_site_builder.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def test_configuration_driven_site_builder(
     site_builder = SiteBuilder(
         data_context=context,
         runtime_environment={"root_directory": context.root_directory},
-        **local_site_config
+        **local_site_config,
```
Comment (AFineDayFor) [inline tests/render/test_data_documentation_site_builder.py]: flake8 had fun here. precommit is love.

**Comment by AFineDayFor** [comment]
```python
In [1]: %load test

In [2]: # %load test
   ...: data_context_id = "f43d4897-385f-4366-82b0-1a8eda2bf79c"
   ...: tz_data = "20190924T231836.000000Z"
   ...: static_logos_dir = "https://company-visual-assets.s3.amazonaws.com/logos"
   ...: static_logo_file = "great-expectations-long-logo.png"
   ...: static_logo_path = f"{static_logos_dir}/{static_logo_file}"
   ...: expected_logo_url = (
   ...:     f"{static_logo_path}?d={tz_data}&dataContextId={data_context_id}"
   ...: )
   ...:

In [3]: expected_logo_url
Out[3]: 'https://company-visual-assets.s3.amazonaws.com/logos/great-expectations-long-logo.png?d=20190924T231836.000000Z&dataContextId=f43d4897-385f-4366-82b0-1a8eda2bf79c'

In [4]:
```

e.g. 
https://company-visual-assets.s3.amazonaws.com/logos/great-expectations-long-logo.png?d=20190924T231836.000000Z&dataContextId=f43d4897-385f-4366-82b0-1a8eda2bf79c

^ click

**Review by cdkini — APPROVED** [review]
Please update the title but LGTM

**Comment by austiezr** [comment]
Closing as overtaken by events; correct logos are now in use.

**Pull request closed without merging**